addons/website_slides_survey/data/mail_template_data.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="mail_template_user_input_certification_failed" model="mail.template">
+            <field name="name">Certification failed email</field>
+            <field name="model_id" ref="model_survey_user_input" />
+            <field name="partner_to">${object.partner_id.id}</field>
+            <field name="subject">You have failed the course: ${object.slide_partner_id.channel_id.name}</field>
```

Comment:
Side note: be sure we won't have issues browsing slide_partner_id due to ACLS (aka, check call stack).

Reply:
The email is sent in a method that is called only on write/create on user_input.
The "submit" method of survey uses a "answer_sudo" to write on it so it should be ok.

The only issue I could see is a survey_user/survey_manager trying to create/write the user_input and manually set its state to "done" but it feels like a weird use case.